exercises/Clustering/notebooks/01_Clustering_Embryoid_Body_Data.ipynb

```diff
@@ -693,7 +700,7 @@
     "\n",
     "In the above exercise, we manually inspected known marker genes to annotate clusters. Here, we're going to simply identify the genes that change the most between two groups. As discussed in lecture, there are several ways to calculate differential expression. We are going to focus on the mean-difference and the rank-sum statistic.\n",
     "\n",
-    "You can easily calculate basic differential expression statistics using `scprep.stats.differential_expression` and `scprep.stats.differential_expression_by_cluster`. Documentation for these methods can be found at: https://scprep.readthedocs.io/en/stable/reference.html#scprep.stats.differential_expression\n",
+    "You can easily calculate basic differential expression statistics using `scprep.stats.differential_expression` and `scprep.stats.differential_expression_by_cluster`. Documentation for these methods can be found at: https://scprep.readthedocs.io/en/stable/reference.html#scprep.stats.differential_expression.\n",
```

Comment:
Adding a period at the end of the URL is risky. Can we fix this with markdown? 
```
[https://scprep.readthedocs.io/en/stable/reference.html#scprep.stats.differential_expression](https://scprep.readthedocs.io/en/stable/reference.html#scprep.stats.differential_expression).
```
renders as [https://scprep.readthedocs.io/en/stable/reference.html#scprep.stats.differential_expression](https://scprep.readthedocs.io/en/stable/reference.html#scprep.stats.differential_expression).